src/Database.pas

```diff
@@ -2911,7 +2913,7 @@ procedure CreateUnit(p, mix: integer);
       Status := 0;
       SavedStatus := 0;
       Inc(Model[mix].Built);
-      Home := -1;
+      Home := 0; //Lagi: -1 is homeless, support free settler
```

Comment:
making settler behave normal (as every other) is "game breaking", but introducing new homeless killing mechanic is a thing to consider?! 
[ no hard feelings, im calm & polite, but suprised ]

food is only in Despotism and code is work  (i tested it), its work as intended (expand code view). this change is to compensate for AI need to support starting settler. You can remove this palace food bonus but it affect AI - they advance slower because lacking 1 food